opensearch_py_ml/sentence_transformer_model/sentencetransformermodel.py

```diff
@@ -743,6 +746,95 @@ def set_up_accelerate_config(
         with open(file_path, "w") as file:
             yaml.dump(default_file, file)
 
+    def make_model_config_json(
+        self,
+        model_name: str = None,
+        version_number: int = 1,
+        embedding_dimension: int = 384,
+        all_config: dict = None,
+        model_type: str = None,
+    ) -> None:
+        """
+        parse from config.json file of pre-trained hugging-face model to generate a model_config.json file. If all required
+        fields are given by users, use the given parameters and will skip reading the config.json,
+
+        Parameters
+        ----------
+        model_name: str = None
+            Optional, The name of the model. If None, default to parse from model id, for example,
+            'msmarco-distilbert-base-tas-b'
+        version_number: int = 1,
+            Optional, The version number of the model. If None, default to be 1.
+        embedding_dimension: int = 384,
+            Optional, the embedding_dimension of the model. If None, parse from the config file of pre-trained
+            hugging-face model, if not found, default to be 384.
+        all_config: dict = None,
+            Optional, the embedding_dimension of the model. If None, parse from the config file of pre-trained
+            hugging-face model.
+        model_type: str = None,
+            Optional, the embedding_dimension of the model. If None, parse from the config file of pre-trained
+            hugging-face model.
+
+        Returns
+        -------
+        None
+        """
+        folder_path = os.path.join(os.getcwd() + "/trained_pytorch_model/")
```

Comment:
the config.json file was saved in this folder during the `train`/ or `save_as_pt`. What happened is that when we download the pretrained model for training or exporting, I saved all the json file in this foler currentpath + "/trained_pytorch_model/". so we should looked it up this folder for 'config.json' file

Reply:
In that case we should make this very clear that, we can use this function only if anybody uses `train` / `save_as_pt` function.

Or we can add a model path parameter, where default value can be: `os.path.join(os.getcwd() + "/trained_pytorch_model/"` ?

Reply:
how about if file not found, then I try to save to this folder again?

Reply:
now I am raising exception if the file not found, 

`if not os.path.exists(config_json_file_path):
                raise Exception(
                    str("Cannot find config.json in" + config_json_file_path)
                )`

when raising exception, we can suggest the user to save the model config file at the folder, then it might be more compatible in the future

Reply:
making the file folder as attribute in the constructor, please see @mingshl
[create folder path in constructor](https://github.com/opensearch-project/opensearch-py-ml/pull/46/commits/7e5570cdd4ee9bb64d1d06c21edb9679bf25bdf6)